api/src/main/java/marquez/db/OpenLineageDao.java

```diff
@@ -540,6 +545,35 @@ default DatasetRecord upsertLineageDataset(
             formatNamespaceName(ds.getNamespace()),
             DEFAULT_NAMESPACE_OWNER);
 
+    DatasetSymlinkRow symlink =
+        datasetSymlinkDao.upsertDatasetSymlinkRow(
+            UUID.randomUUID(),
+            formatDatasetName(ds.getName()),
+            dsNamespace.getUuid(),
+            true,
+            "",
```

Comment:
Similar comment as above, I would avoid using an empty string for the symlink type and default to `UNKNOWN` or `null`.

Reply:
let's go with `null`